### ISSUE TITLE
Need commercial support!

### ISSUE THREAD

**Issue by ghost** [mattdiamond/fuckitjs#22]
Good piece of software, needed something like this for years. Do you offer commercial support?

**Comment by kofalt** [comment]
![jvm](https://cloud.githubusercontent.com/assets/265071/2783302/3ef175f6-cb2b-11e3-862c-7766319e0871.gif)

**Comment by mattdiamond** [comment]
Definitely. For the low price of 100k/year, I offer "enterprise support" which guarantees I will reply to your emails within 4-6 weeks of noticing them.

I accept credit card, bitcoin, and briefcases full of cash.

**Comment by syzer** [comment]
is this technology enterprise ready? :)

**Comment by jberger** [comment]
I suspect that this is a fairly accurate representation of most enterprise ready software
